Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "UnitfulIntegration"
+uuid = "d000b014-1cca-4823-a42a-b615c320b663"
+authors = ["contributors"]
```

Comment:
```suggestion
```
IMO, just writing "contributors" doesn’t add much value. I would just remove the line (the `authors` field is not required, Unitful doesn’t have one either).